project_task_pull_request_state/tests/test_project_task_pull_request_state.py

```diff
@@ -0,0 +1,175 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo.tests.common import TransactionCase
+
+
+class TestPullRequestState(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestPullRequestState, cls).setUpClass()
```

Comment:
Disable tracking unless you need it for functional purposes

    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))

Reply:
resolved